src/components/account/Account.js

```diff
@@ -9,12 +9,42 @@ import Paper from '@mui/material/Paper';
 import Header from '../Header/Header';
 import './style.css';
 import EditDetails from './EditDetails';
+import client from '../../utils/client';
 
 function createData(key, value) {
   return { key, value };
 }
 
-const Account = ({ user }) => {
+const Account = ({ getLoggedInUserId, user, setUser }) => {
+  const { email } = user
+  const reqBodyEmail = email
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    const reqBody = { email: reqBodyEmail }
```

Comment:
You don't need the `reqBodyEmail` constant since you've already destructured the email into its own variable called `email`, you can change this to be `const reqBody = { email }`